server/src/main/java/fr/uga/l3miage/tp2/exo3/moduls/CategorieDeProduit.java

```diff
@@ -0,0 +1,16 @@
+package fr.uga.l3miage.tp2.exo3.moduls;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class CategorieDeProduit {
+    @Id
+    private Long id;
+
+    private String nom;
+
+    @ManyToOne
+    private Produit produit;
```

Comment:
```suggestion
    @OneToMany(mappedBy="categorieDeProduit")
    private Set<Produit> produitEntities;
```

Dommage vous vous êtes tromper dans la relation entre `produit` et `categorie`